Source/Core/Core/FifoPlayer/FifoAnalyzer.cpp

```diff
@@ -247,9 +270,18 @@ u32 AnalyzeCommand(const u8* data, DecodeMode mode)
           FifoRecordAnalyzer::WriteVertexArray(static_cast<int>(i), data + offsets[i], vertexSize,
                                                numVertices);
         }
+
+        auto [iter, inserted] = hash_to_entry.try_emplace(Common::GetHash64(data, datasize, 0),
```

Comment:
The VAT format (and anything else `CalculateVertexElementSizes` reads) really should be incorporated into this hash.